### DISCUSSION TITLE
Larger area to click and organize xsheet columns

### DISCUSSION THREAD

**Discussion by age3rcm** [opentoonz/opentoonz#6821] [Ideas]
**Is your feature request related to a problem? Please describe**
many times, i miss the click to order columns, the area is very small

**Describe the solution you'd like**
as you see on the image, you get a designated area wich has more space without impeding the normal operation
![unknown (3)](https://user-images.githubusercontent.com/36406840/111889244-8a301600-89c2-11eb-9c66-af5bcaab8196.png)

**Comment by DarrenTAnims** [comment]
You should be able to click and drag anywhere on the header bar to re-order the columns. It's what I'd expect and as a user, I can't see a huge problem why you couldn't. The only thing, potentially, is that you don't want to click to select a column and have the column in drag mode, but plenty of other software allows a single click on an object to select, but also making it draggable.

**Comment by artisteacher** [comment]
It can be easy to start a selection instead of beginning a drag/drop action. I've wondered if the target area is very slightly off. If you aim for the top of the dragbar instead of the bottom, drag/drop is much more reliable.

Clicking and dragging from here will start a selection - even though the tip of the mouse is on the dragbar:
<img width="63" alt="Screen Shot 2021-03-21 at 10 03 54 AM" src="https://user-images.githubusercontent.com/21159116/111907703-e7f94800-8a2c-11eb-95de-a153b2cfadd0.png">
Aim any higher, and you should be able to trigger drag/drop without issue.

**Comment by age3rcm** [comment]
>  I can't see a huge problem why you couldn't. The only thing, potentially, is that you don't want to click to select a column and have the column in drag mode, 

considering its an small area, it will be easier to do and more acessible to have a bigger area to do that
